package.json

```
@@ -43,6 +43,7 @@
     "camelcase": "7.0.0",
     "chalk-template": "0.4.0",
     "chokidar": "3.5.3",
+    "compression": "^1.7.4",
```

we usually lock the version, so that there is no surprise when installing the tool.
(although, nowadays, the packages follow semver more strictly than 4-5 years ago).

```suggestion
    "compression": "1.7.4",
```

---

Is that a net-new dependency or just something we turned from transitive to direct?